<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="ChatEdit">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>520</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="MessageSendButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>520</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>전송</string>
    </property>
   </widget>
   <widget class="QListView" name="FriendsList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>261</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="FreindSearch">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>검색</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>친구명</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FileSendButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>520</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>파일 전송</string>
    </property>
   </widget>
   <widget class="QListWidget" name="ChattingList">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>50</y>
      <width>311</width>
      <height>461</height>
     </rect>
    </property>
    <property name="autoScrollMargin">
     <number>100</number>
    </property>
    <property name="alternatingRowColors">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>664</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
